arcgis_map_sdk_ios/ios/arcgis_map_sdk_ios/Sources/arcgis_map_sdk_ios/ManualLocationDataSource.swift

```diff
@@ -8,15 +8,38 @@
 import Foundation
 import ArcGIS
 
-class ManualLocationDataSource: AGSLocationDataSource {
+final class CustomLocationProvider: LocationProvider {
```

Comment:
I did some testing and the location indicator disappears when I switch the location source to manual. I enabled the pan mode to be navigation, so I expected the map to move to the position. 🤔